test/samlp.tests.js

```diff
@@ -670,6 +678,20 @@ describe('samlp', function () {
           done();
         });
       });
+
+      describe('when invalid signing key is used', function () {
+        before(function () {
```

Comment:
Do you need to undo this in `after`?

Reply:
Good point! I've added `after` to clean up after the test execution, even though it was not a reason why tests were failing (see the comment below).